sshcommand

```diff
@@ -58,6 +58,13 @@ case "$1" in
 
     getent passwd "$USER" > /dev/null || false
     USERHOME=$(sh -c "echo ~$USER")
+
+    NEW_KEY=$(grep "NAME=\\\\\"$NAME"\\\\\" "$USERHOME/.ssh/authorized_keys" || true)
+    if [ ! -z "$NEW_KEY" ]; then
+      echo "Duplicated SSH Key name"
```

Comment:
Should probably say "Duplicate". @josegonzalez mentioned printing this to stderr but we don't do that pattern anywhere else. @josegonzalez what was your thinking there?